### ISSUE TITLE
Sequence Diagrams: Compliance with code

### ISSUE THREAD

**Issue by qweiping31415** [nus-cs2103-AY1920S1/forum#197]
The question here is quoted from the sample paper.

![Picture3](https://user-images.githubusercontent.com/38523207/69785329-c0a4b900-11f2-11ea-98f8-983c2c846295.png)

A26 answer is given to be (A). Is this because the code does not show any inconsistencies with the sequence diagram, even though it shows no information about the p.up() line that should be there in the code?

**Comment by damithc** [comment]
> Is this because the code does not show any inconsistencies with the sequence diagram, even though it shows no information about the p.up() line that should be there in the code?

Yes. Note the `//some more code here` which could be the code that contains `p.up()` call.
